collect_app/src/main/java/org/odk/collect/android/widgets/AudioWidget.java

```diff
@@ -144,40 +144,25 @@ public IAnswerData getAnswer() {
         }
     }
 
-    /**
```

Comment:
`FormSaveViewModel.createAnswerFile` returns a `LiveData<Result<String>>` used to set the data from a file name. Have you tried this change with internal recording?

Reply:
Interesting! @seadowg, would be good for you to at least see this change happening since it's to very recent audio code.

Reply:
Looks like @lognaturel is right as `AudioRecordingTest` and `AudioWidgetTest` are failing. From debugging, it looks like `AudioWidget#setData` is still passed a `String`.

Reply:
Do you remember whether there was a specific reason you wanted to use a `String`, @seadowg? Looks like using a `LiveData<Result<File>>` works fine but want to make sure there isn't a strong reason against it.

Reply:
Yeah I have just turned it into files so please answer if there was any reason.

Reply:
I don't **think** it needs to be a `String`. It was just because it didn't need to be a file as the the other side could just use `getAnswerFile` with the name (the `String`) to access the actual file. I think if `File` is more consistent we can go with that!